src/guru.jl

```diff
@@ -170,10 +169,9 @@ function finufft_setpts(plan::finufft_plan{T},
                       Ref{Cdouble},
                       Ref{Cdouble},
                       Ref{Cdouble}),
-                     plan.plan_ptr,M,T_real(xj),T_real(yj),T_real(zj),N,T_real(s),T_real(t),T_real(u)
+                     plan.plan_ptr, M, xj, yj, zj, N, s, t, u
```

Comment:
makes sense to simplify, but I wonder why Libin put in the conversions in the first place?

Reply:
> makes sense to simplify, but I wonder why Libin put in the conversions in the first place?

I think I was trying to do a array sanity check as in python interface, https://github.com/flatironinstitute/finufft/blob/4fe380544202f0ad7b89f52c4e08a70f090ed22b/python/finufft/_interfaces.py#L175

While here in Julia we enforce the type check in function signature, so the type conversion is not necessary.